appendix-keyboard.tex

```diff
@@ -151,7 +151,7 @@ \section{Keyboard LED Control}
 The LEDs on the MEGA65's keyboard are normally controlled automatically
 by the system.  However, it is also possible to place them under
 user control.  This is activated by setting bit 7 of \$D61D (decimal 54813).
-The lower bits indicate which keyboard register to set.  Registers 0 through 11
+The lower bits indicate which keyboard LED register to set.  Registers 0 through 11
 correspond to the red, green and blue channels of the four LEDs.  The intensity

```

Comment:
Wait ... the four LEDs ... are those Power LED, Drive LED, Shift-Lock LED & Caps-Lock LED ... and each of those is an RGB LED ?

Reply:
a. No, the four LEDs are because the power and drive LEDs are actually pairs of LEDs. We should make this clear.
b. The $\leftarrow$ that was not in a \megakey{} in the matix is the key next to 1, and was not put in a key box to distinguish it from the cursor left key. So we still need some way to differentiate them.  Maybe have a think about the best way to do that, or alternatively, we can add a footnote to the bottom of the matrix table.

Otherwise looks great, and thanks for the contibution!

Reply:
> b. The $\leftarrow$ that was not in a \megakey{} in the matix is the key next to 1, and was not put in a key box to distinguish it from the cursor left key. So we still need some way to differentiate them.

Fixed by e62d93cbe0b3b0e05bb6ae6f26659dc4dbf13410

Reply:
> a. No, the four LEDs are because the power and drive LEDs are actually pairs of LEDs. We should make this clear.

How about 5706b7b847efbad2642a54210d18ef1e328792d1 ?